src/utils/assetManagers/assetManagers.test.js

```diff
@@ -271,4 +271,27 @@ describe('utils/assetManagers', () => {
       })
     })
   })
+
+  describe('retrieveEODBooks', () => {
```

Comment:
You should also amend `Promise.resolve(data)` to resolve with some form of mocked EOD book. E.g.

```javascript
const mockedEODBook = {
  assetManagerId: 1,
  utcCloseTime: '18:00:00',
  bookId: 'testID',
  eodBookStatus: 'testStatus'
}
// ...
beforeAll(() => {
  network.retrieveData.mockImplementation(() => Promise.resolve(mockedEODBook)
})
// ...
```